FWCore/Services/plugins/DependencyGraph.cc

```diff
@@ -222,7 +222,7 @@ void DependencyGraph::preBeginJob(PathsAndConsumesOfModulesBase const &pathsAndC
     boost::get_property(m_graph, boost::graph_graph_attribute)["labelloc"] = "top";
 
     // create graph vertices associated to all modules in the process
-    auto size = pathsAndConsumes.allModules().size();
+    auto size = pathsAndConsumes.largestModuleID() - boost::num_vertices(m_graph) + 1;
```

Comment:
@fwyzard Subtracting the number of already existing vertices from the largest module ID is kind of closest to earlier "number of modules", and is needed make the SubProcesses to have proper boundaries in the graph. Should the `ProcessCallGraph` be updated accordingly?

Here is an example how the deleted modules get treated
![image](https://user-images.githubusercontent.com/33993/83768288-32e3d380-a644-11ea-863b-a9a9a09cb376.png)
i.e. they are presented as ellipses with a number